src/documents/activities/StatesandContextsMapping.html.md.handlebars

```diff
@@ -0,0 +1,42 @@
+---
+title: States and Contexts Mapping
+layout: content
+category: Activities
+---
+
+This activity breaks down the User States and Contexts tool into modular pieces and actionable steps, and it is also a re-thinking of the original User States and Contexts tool.
```

Comment:
Maybe split the part about the "re-thinking" into a separate sentence and put a link to the original US&C tool?